package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@aragon/osx-commons-repo",
+  "version": "1.0.0",
```

Comment:
see https://discord.com/channels/672466989217873929/1173682621964877884

What version number should we pick? Is there a better way?